### ISSUE TITLE
Table(React) fails axe check when using currentSort prop

### ISSUE THREAD

**Issue by andresriveratoro** [department-of-veterans-affairs/va.gov-team#36910]
## Issue Description
Our React Table component is failing an accessibility check in `vets-website` when using the currentSort prop. See https://github.com/department-of-veterans-affairs/vets-website/runs/5189913495?check_suite_focus=true ([Pull Request](https://github.com/department-of-veterans-affairs/vets-website/pull/20223))

## Acceptance Criteria
- [ ] Table React component passes axe check using `axe-core^4.3.5`
---

**Comment by andresriveratoro** [comment]
Possible solution (thanks @bkjohnson):
<img width="833" alt="Screen Shot 2022-02-14 at 18 30 26" src="https://user-images.githubusercontent.com/36863582/153964945-4dec2b59-a157-461e-98c1-671b6afc4865.png">
